lib/calendar_api.py

```diff
@@ -1,20 +1,21 @@
 import os.path
-
 from google.auth.transport.requests import Request
 from google.oauth2.credentials import Credentials
 from google_auth_oauthlib.flow import InstalledAppFlow
 from googleapiclient.discovery import build
 from googleapiclient.errors import HttpError
+from .entity.event import Event
+from dotenv import load_dotenv
 
-from .vtools_fetcher import Event
+load_dotenv()
```

Comment:
WDYT about adding `load_dotenv()` in `main.py` instead of downstream `lib/` scripts. It might help to decouple the responsibilities of these.

I'm thinking on making `CALENDAR_ID` a constructor parameter of CalendarAPI

Reply:
I agree, and it's the same for vTools. Prioritize as an interim measure so as not to block other users from using it early. Let's create a new Issue to deal with that issue specifically.